wallet/src/de/schildbach/wallet/ui/SettingsActivity.kt

```diff
@@ -88,17 +87,14 @@ class SettingsActivity : BaseMenuActivity() {
             getString(R.string.preferences_initiate_reset_dialog_positive)
         ).show(this) {
             if (it == true) {
-                isFinished = true
                 log.info("manually initiated blockchain reset")
                 analytics.logEvent(AnalyticsConstants.Settings.RESCAN_BLOCKCHAIN_RESET, bundleOf())
 
                 WalletApplication.getInstance().resetBlockchain()
                 WalletApplication.getInstance().configuration.updateLastBlockchainResetTime()
                 startActivity(WalletActivity.createIntent(this@SettingsActivity))
             } else {
-                if (!isFinished) {
```

Comment:
Doesn't looks like `isFinished` is needed. Might be a leftover from previous refactoring.